test/vre/chapter36/config/applicantInformation.unit.spec.jsx

```diff
@@ -50,8 +58,9 @@ describe('VRE chapter 36 applicant information', () => {
     expect(form.find('.usa-input-error').length).to.equal(1);
     expect(onSubmit.called).to.be.false;
   });
+  */
 
-  it('should not submit without required applicant info', () => {
+  it.only('should not submit without required applicant info', () => {
```

Comment:
Need some guidance on what's being tested here and how to set it up in the form. Currently I can continue without any of the questions answered. Are required fields necessary before **Continue** will work or before **Submit** works on the review page?

Reply:
They are necessary in both cases. I think the confusing here is partly what I mentioned a second ago, that you're building another page, not modifying the first one.

Reply:
That makes sense. Looks like the test files in hca are organized around these pages so I'll move these to another file and restore the deleted tests.